api/room/schema_query.py

```diff
@@ -431,15 +431,19 @@ def resolve_bookings_analytics_count(
             self, query, start_date, end_date, room_id=room_id)
         return analytics
 
+    @Auth.user_roles('Admin', 'Default User')
     def resolve_analytics_for_daily_room_events(
-        self, info, start_date, end_date, page=None, per_page=None
+        self, info, **kwargs
     ):
+        start_date = kwargs.get('start_date')
+        end_date = kwargs.get('end_date')
+        location_id = admin_roles.user_location_for_analytics_view()
         start_date, end_date = CommonAnalytics().convert_dates(
```

Comment:
Nice work on the PR @daniellamarr 
There is a bit on ambiguity with some variable names. The `start_date` and `end_date` appeared multiple times in a manner that confuses me. Can you please explain to me what's happening in this section of the code as regards to `start_date` and `end_date`?